requirements.txt

```diff
@@ -0,0 +1,11 @@
+certifi==2021.5.30
```

Comment:
Why separate requirements.txt when setup.py also defines dependencies?

Reply:
based on python best practises: https://packaging.python.org/discussions/install-requires-vs-requirements/ there is good to have "minimum versions" defined in `install_requires` field in `setup.py` and pinned versions in `requirements.txt`..

> Whereas install_requires requirements are minimal, requirements files often contain an exhaustive listing of pinned versions for the purpose of achieving repeatable installations of a complete environment.